src/Toggle/ToggleChecker.php

```diff
@@ -9,6 +9,7 @@
 
 class ToggleChecker implements IsToggleActive
 {
+    const TOGGLE_TYPE_GROUP = 'group';
```

Comment:
I don't know where would be the best place for these constants. Right now I have the same here:
https://github.com/clearbooks/labs-php-client-mysql/blob/is-toggle-active/src/Db/Table/Toggle.php#L8

But obviously we can't use that either here.